src/Pages/RandomPhoto.js

```diff
@@ -0,0 +1,91 @@
+import React, { useState, useEffect } from "react";
```

Comment:
This component is also pretty similar to the others, i think you can make 1 generic component that handles your 3 useCases